__tests__/app.test.js

```diff
@@ -25,4 +25,31 @@ describe("GET /api/categories", () => {
             })
         })
     })
+})
+
+describe("GET /api/reviews", () => {
+    test("200: returns an array of objects, including comment_count and sorted by descending order", () => {
+        return request(app)
+        .get("/api/reviews")
+        .expect(200)
+        .then(( { body } ) => {
+            const reviews = body.reviews
+            expect(reviews).toBeInstanceOf(Array)
+            expect(reviews).toHaveLength(13)
+            expect(reviews).toBeSorted('created_at', { descending: true })
```

Comment:
Great use of the sortedBy package to check these are being returned in the correct order 👍